qa/src/test/java/io/camunda/migrator/qa/distribution/DistributionSmokeTest.java

```diff
@@ -0,0 +1,275 @@
+    assertThat(output).contains("--runtime");
+    assertThat(output).contains("--history");
+    assertThat(output).contains("--list-skipped");
+    assertThat(output).contains("--retry-skipped");
+  }
+
+  @Test
+  @Timeout(value = 60, unit = TimeUnit.SECONDS)
+  void shouldShowUsageWhenTooManyArgumentsProvided() throws Exception {
+    // given
+    ProcessBuilder processBuilder = new ProcessBuilder("./start.sh", "--runtime", "--history", "--list-skipped", "--retry-skipped");
+    processBuilder.directory(extractedDistributionPath.toFile());
+    processBuilder.redirectErrorStream(true);
+
+    // when
+    Process process = processBuilder.start();
+
+    // then
+    String output = readProcessOutput(process);
+    int exitCode = process.waitFor();
+
+    assertThat(exitCode).isEqualTo(1);
+    assertThat(output).contains("Error: Too many arguments.");
+    assertThat(output).contains("Usage: start.sh");
+  }
+
+  @Test
+  @Timeout(value = 60, unit = TimeUnit.SECONDS)
+  void shouldAcceptValidFlags() throws Exception {
+    // given
+    String[] validFlags = {"--runtime", "--history", "--list-skipped", "--retry-skipped"};
+
+    for (String flag : validFlags) {
+      ProcessBuilder processBuilder = new ProcessBuilder("./start.sh", flag);
+      processBuilder.directory(extractedDistributionPath.toFile());
+      processBuilder.redirectErrorStream(true);
+
+      // when
+      process = processBuilder.start();
+
+      // then
+      String output = readProcessOutput(process);
+
+      assertThat(output).contains("Starting migration with flags: " + flag);
+    }
+  }
+
+  @Test
+  @Timeout(value = 30, unit = TimeUnit.SECONDS)
+  void shouldStartWithoutArgumentsAndShowExpectedMessage() throws Exception {
+    // given
+    ProcessBuilder processBuilder = new ProcessBuilder("./start.sh");
+    processBuilder.directory(extractedDistributionPath.toFile());
+    processBuilder.redirectErrorStream(true);
+
+    // when
+    process = processBuilder.start();
+
+    // then
+    String output = readProcessOutput(process);
+    assertThat(output).contains("Starting application without migration flags");
+  }
+
+  @Test
+  void shouldHaveRequiredFilesInDistribution() {
+    assertThat(extractedDistributionPath.resolve("start.sh")).exists();
+    assertThat(extractedDistributionPath.resolve("start.bat")).exists();
+    assertThat(extractedDistributionPath.resolve("configuration")).exists();
+    assertThat(extractedDistributionPath.resolve("internal")).exists();
+    assertThat(extractedDistributionPath.resolve("LICENSE.TXT")).exists();
+    assertThat(extractedDistributionPath.resolve("NOTICE.txt")).exists();
+    assertThat(extractedDistributionPath.resolve("README.txt")).exists();
+  }
+
+  @Test
+  void shouldHaveExecutableStartScript() {
+    assertThat(startScriptPath.toFile().canExecute()).isTrue();
+  }
+
+  @Test
+  @Timeout(value = 30, unit = TimeUnit.SECONDS)
+  void shouldDeployBpmnModelFromResourcesFolder() throws Exception {
```

Comment:
🙃 IMHO this change would help to understand why is a failure log is expected in the tests. 
```suggestion
  void shouldTryToDeployBpmnModelFromResourcesFolder() throws Exception {
```